.expeditor/config.yml

```diff
@@ -165,7 +165,7 @@ subscriptions:
     - built_in:promote_habitat_packages
     - bash:.expeditor/publish-release-notes.sh:
        post_commit: true
-    - bash:.expeditor/purge-cdn.sh:
+    - purge_packages_chef_io_fastly:{{target_channel}}/inspec/latest
```

Comment:
so `artifact_published` doesn't have a `target_channel` metadata. This should likely be just `{{channel}}` or hardcoded to `stable`:  https://expeditor-acceptance.chef.io/docs/reference/workloads/#artifact_published